src/routes/github/webhook/webhook-receiver-post.ts

```diff
@@ -0,0 +1,78 @@
+import Logger from "bunyan";
+import { BinaryLike, createHmac } from "crypto";
+import { Request, Response } from "express";
+import { validationResult } from "express-validator";
+import { getLogger } from "~/src/config/logger";
+import { GitHubServerApp } from "~/src/models/git-hub-server-app";
+import { WebhookContext } from "./webhook-context";
+
+export const WebhookReceiverPost = async (request: Request, response: Response): Promise<void> => {
+	const errors = validationResult(request);
+	if (!errors.isEmpty()) {
+		response.status(400).json({ errors: errors.array() });
+		return;
+	}
+	const logger = getLogger("webhook.receiver");
+	const eventName = request.headers["x-github-event"] as string;
+	const signatureSHA256 = request.headers["x-hub-signature-256"] as string;
+	const id = request.headers["x-github-delivery"] as string;
```

Comment:
aren't all headers already strings?

Reply:
It is of type `string  |  string[]  |  undefined`